content/docs/user-guide/experiment-management/index.md

```diff
@@ -1,110 +0,0 @@
-# Experiment Management
-
-Data science and machine learning are iterative processes that require a large
-number of attempts to reach a certain level of a metric. Experimentation is part
-of the development of data features, hyperspace exploration, deep learning
-optimization, etc.
-
-Some of DVC's base features already help you codify and analyze experiments.
-[Parameters](/doc/command-reference/params) are values in a structured text
-file, which you can tweak and use in your code. On the other end,
-[metrics](/doc/command-reference/metrics) (and
-[plots](/doc/command-reference/plots)) let you define, visualize, and compare
-quantitative measures of your results.
-
-## Experimentation in DVC
-
-DVC experiment management features conform a comprehensive framework to
-organize, execute, manage, and share ML experiments. They support these main
-approaches:
-
-- Compare parameters and metrics of existing project versions (for example Git
-  branches) against each other or against new results in the workspace. See for
-  example `dvc exp diff`.
-
-- [Run and capture] multiple experiments (derived from any project version as
-  baseline) without polluting your Git history. DVC tracks them for you, letting
-  you compare and share them.
-
-  <admon icon="book">
-
-  More info in the [Experiments Overview][experiments].
-
-  </admon>
-
-  <admon type="tip">
-
-  **New!** DVC Experiments can be used directly [from the VS Code IDE] or online
-  with [Iterative Studio], the web UI that integrates all of our data science
-  tools.
-
-  </admon>
-
-[from the vs code ide]: /doc/vs-code-extension
-[iterative studio]: /doc/studio
-
-- Generate [checkpoints] at runtime to keep track of the internal progress of
-  deeper experiments. DVC captures [live metrics](/doc/dvclive), which you can
-  manage in batches.
-
-[run and capture]: /doc/user-guide/experiment-management/running-experiments
-[experiments]: /doc/user-guide/experiment-management/experiments-overview
-[checkpoints]: /doc/user-guide/experiment-management/checkpoints
-
-<admon type="">
-
-See 👨‍💻 [Get Started: Experiments] for a hands-on introduction to DVC
-experiments.
-
-</admon>
-
-[get started: experiments]: /doc/start/experiment-management/experiments
```

Comment:
Maybe this one too

Reply:
This get started page is pretty outdated, but it still probably makes sense to include it, and hopefully we will get around to updating it soon. Added it back.